testsuite/.rubocop_todo.yml

```diff
@@ -259,30 +260,31 @@ Lint/UnusedBlockArgument:
   Exclude:
     - 'ext-tools/maintenance_crawler.rb'
 
-# Offense count: 14
+# Offense count: 17
 Lint/UselessAssignment:
   Exclude:
     - 'features/step_definitions/command_steps.rb'
     - 'features/step_definitions/common_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
+    - 'features/step_definitions/vm_steps.rb'
```

Comment:
I know it's out of topic, but if you can make it in a way to avoid adding this extra exclusion, fixing whatever Rubocop don't like... that would be awesome.

Reply:
Fixed.